Source/DataSources/Angle.js

```diff
@@ -0,0 +1,97 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/Event'
+    ], function(
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        Event) {
+    "use strict";
+
+    /**
+     * A property type whose value is always interpolated by taking the the shortest distances between 2 angles.
```

Comment:
The documentation is incorrect here.  This object is not a property, I would refer to it as a Packable Number, and link to the Packable interface.  I would also  mention that it is not used on it's own but in conjunction with `SampledProperty` and add an `@example` to show it's usage.